app/views/flood_risk_engine/enrollments/steps/check_your_answers.html.erb

```diff
@@ -4,50 +4,38 @@
              as: step,
              url: enrollment_step_path(form.enrollment, step),
              method: :patch,
+             builder: GOVUKDesignSystemFormBuilder::FormBuilder,
              class: 'form' do |f| %>
 
-  <%= render(
-          "flood_risk_engine/enrollments/header", title_text: t("flood_risk_engine.enrollments.steps.#{step}.heading")
-      )
-  %>
-
-  <h2 class="heading-medium">
+  <h1 class="govuk-heading-l">
+    <%= t("flood_risk_engine.enrollments.steps.#{step}.heading") %>
+  </h1>
+  <h2 class="govuk-heading-m">
     <%= t(".table_heading") %>
   </h2>
-
-  <table class="check-your-answers">
-    <caption class="visuallyhidden"><%= t('.table_summary') %></caption>
-    <tbody>
-      <% form.rows.each do |row| %>
-        <tr data-row="<%= row.name %>">
-          <th>
-            <%= row.title %>
-          </th>
-          <td>
-            <%= row.value %>
-          </td>
-          <td class="change-answer">
-            <% include_change_link = false # remove this when we do the story about adding change links %>
```

Comment:
Are we enabling change links as part of this story? If so I think this needs more discussion before we make the change here, as it will make the workflow a lot more complicated and need some additional work to get that going.

Reply:
I stripped back the change links behaviour, thinking that we could tackle it later

Reply:
Cool - could we maybe leave a comment on RUBY-1464 just so we're aware we need to deal with it there?